src/Adapter/MySQL.php

```diff
@@ -336,9 +339,71 @@ private function computeOrderByField(array $filterToTableMapping)
             $orderField = 'p.' . $orderField;
         }
 
+        // put some products at the end of the list
+        $orderField = $this->computeShowLast($orderField);
+
         return $orderField;
     }
 
+    /**
+     * Sort product list: InStock, OOPS with qty 0, OutOfStock
+     * @param  string $orderField
+     * @return string
+     */
+    private function computeShowLast($orderField)
+    {
+        // allow only if feature is enabled & it is main product list query
+        if ($this->getInitialPopulation() === null
+            || empty($orderField)
+            || ! \Configuration::get('PS_LAYERED_FILTER_SHOW_OUT_OF_STOCK_LAST')
```

Comment:
Use namespace instead
```suggestion
            || !Configuration::get('PS_LAYERED_FILTER_SHOW_OUT_OF_STOCK_LAST')
```

Reply:
Done. Commit: https://github.com/PrestaShop/ps_facetedsearch/pull/154/commits/69e877bffac6fd00b4ff88a0dd1e907cb0c75f18